school_attendance/__manifest__.py

```diff
@@ -0,0 +1,26 @@
+# See LICENSE file for full copyright and licensing details.
+
+{
+    "name": "Attendance Management",
```

Comment:
Why naming is important?
- So it does not confuse users with HR attendance.
- Make smart name like Attendance Management for Education ERP

Thanks.